lib/common.sh

```diff
@@ -406,6 +406,9 @@ function init_minikube() {
 
     # Restart libvirtd
     sudo systemctl restart libvirtd
+
+    # Wait a moment for libvirtd to come up before continuing with other commands
+    sleep 2
```

Comment:
This seems a bit hackish, do we have a away to tell minikube to try again for wait longer for libvirt?

Reply:
The problem is that there's a timeout that's too short in minikube. I've raised an issue and submitted a patch: https://github.com/kubernetes/minikube/issues/11113. Until this is merged and a new version of minikube is released, we'll need to `sleep` here.

Reply:
The other alternative is to delete the `sudo systemctl restart libvirtd` line. But this PR suggests that it's necessary to fix a networking issue? https://github.com/metal3-io/metal3-dev-env/pull/575

Reply:
I don't recall the actual problem which intrigued the restart, you can try to remove it to test if it solve the issue @macaptain

Reply:
@kashifest I didn't run into any problems when I removed the libvirtd in my metal3-dev-env, so let's try removing it instead of sleeping.